packages/google_maps_flutter/google_maps_flutter_platform_interface/lib/src/platform_interface/google_maps_flutter_platform.dart

```diff
@@ -360,6 +360,11 @@ abstract class GoogleMapsFlutterPlatform extends PlatformInterface {
     throw UnimplementedError('onLongPress() has not been implemented.');
   }
 
+  /// A Map has been tapped on a POI [LatLng, String, String].
+  Stream<MapPoiClickEvent> onPoiClick({required int mapId}) {
+    throw UnimplementedError('onPoiClick() has not been implemented.');
```

Comment:
You are adding unconditional calls to this new method, which will immediately crash anyone who is using a third-party implementation. If the app-facing package is going to use this unconditionally, then the default implementation needs to be to return a stream that never has any events.

Reply:
I have changed it to ` return const Stream<MapPointOfInterestTapEvent>.empty();`